.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Issue: \U0001F4DC Documentation "
+about: Describe this issue template's purpose here.
```

Comment:
Replace the `about:` boiler plate with something like `Report an issue related to https://pytorch.org/docs`